assets/js/components/Math.js

```diff
@@ -37,100 +37,35 @@ module.exports = class Math {
-                return !!navigator.userAgent.match(/iPad|iPhone/);
-              };
-
-              // Don't scale up if any the following applies:
-              return !(
-
-                // IE
-                Browser.isMSIE ||
-
-                // Safari or Chrome on a Mac
-                (Browser.isMac && (Browser.isSafari || Browser.isChrome)) ||
-
-                // Safari or Chrome on iOS
-                (isProbablyIOS() && Browser.isSafari) ||
-
-                // Aiming to target Firefox on Linux & mobile
-                (Browser.isFirefox && (!(Browser.isMac || Browser.isPC))) ||
-
-                // Some smaller browsers e.g. Brave are identified as "Unknown" . (Although both
-                // Brave and Yandex browsers are based on Chromium and use Blink rendering, Yandex
-                // identifies as "Chrome".)
-                Browser.toString() === 'Unknown'
-              );
-            }
-
-            return shouldBeScaledUp(window.MathJax.Hub.Browser) ? upScaling : basicScaling;
-          }())
-        }
+      loader: {
+        load: ['ui/menu']
       }
-    );
-    this.currentClientWidth = document.body.clientWidth;
-    let resizeTimeout;
-    let resizeThrottler = function resizeThrottler() {
-      if (!resizeTimeout) {
-        resizeTimeout = setTimeout(function () {
-          resizeTimeout = null;
+    };
 
-          if (!!this.window.MathJax && this.currentClientWidth !== document.body.clientWidth) {
-            this.currentClientWidth = document.body.clientWidth;
-            this.window.MathJax.Hub.Queue(['Rerender', this.window.MathJax.Hub]);
-          }
-        }, 300);
+    window.MathJax = {
+      chtml: {
+        scale: 1.04,                  // global scaling factor for all expressions
+        minScale: 1,                  // smallest scaling factor to use
+        matchFontHeight: true,        // true to match ex-height of surrounding font
+        mtextInheritFont: false,       // true to make mtext elements use surrounding font
+        merrorInheritFont: false,      // true to make merror text use surrounding font
+        mtextFont: '',                 // font to use for mtext, if not inheriting (empty means use MathJax fonts)
+        merrorFont: 'serif',           // font to use for merror, if not inheriting (empty means use MathJax fonts)
+        mathmlSpacing: false,          // true for MathML spacing rules, false for TeX rules
+        skipAttributes: {},           // RFDa and other attributes NOT to copy to the outputß
+        displayIndent: '0',           // default for indentshift when set to 'auto'
+        cssStyles: null
       }
     };
-
-    window.addEventListener('resize', resizeThrottler);
   }
 
   static load(doc) {
     let script = doc.createElement('script');
+    script.id = 'MathJax-script';
+    script.async = true;
     script.type = 'text/javascript';
-    script.addEventListener('load', Math.setupResizeHandler);
```

Comment:
This could that has been removed was necessary presumably because the MathML need to be re-rendered each time the viewport changed. Does this need to be re-instated?